packages/react-card/etc/react-card.api.md

```diff
@@ -23,6 +23,7 @@ export const cardClassNames: SlotClassNames<CardSlots>;
 // @public (undocumented)
 export type CardCommons = {
     appearance: 'filled' | 'filled-alternative' | 'outline' | 'subtle';
+    focusable: boolean | 'noTab' | 'tabExit' | 'tabOnly';
```

Comment:
have you considered aligning the values with tabster?

Reply:
at least we should document the options properly, as users might not understand which one is doing what

Reply:
everything is documented in the spec https://github.com/microsoft/fluentui/blob/master/packages/react-card/Spec.md#api

Im fan of documentation living with code though, going forward we should probably thing of ways how to make this DRYer experience

Reply:
> have you considered aligning the values with tabster?

can you point us to those values ? I'm unable to find anything in the docs 🙏  https://github.com/microsoft/tabster/blob/master/README.md

Reply:
As @Hotell mentioned, these are described in the spec. The reason we went with this change was for more humanly readable values as tabster's aren't as self-descriptive.
I can add some JSDoc to `CardCommons` to reflect what was written on the Spec. If you agree, do you propose any specific format, or should I just copy what's written in the Spec?

Reply:
I think descriptions in the "`focusable` property" section are reasonable. We need to have them in the storybook and ideally in jsdoc for developers. I don't think anybody would refer to a spec when using the component.